qcschema/dev/properties/cc_properties.py

```diff
@@ -0,0 +1,45 @@
+"""
+The complete list of Coupled-Cluster properties.
+"""
+
+cc_properties = {}
+
+cc_properties["ccsd_same_spin_correlation_energy"] = {
+    "type":
+    "number",
+    "description":
+    "The unscaled portion of the CCSD correlation energy from same-spin or triplet doubles correlations."
+}
+
+cc_properties["ccsd_opposite_spin_correlation_energy"] = {
+    "type":
+    "number",
+    "description":
+    "The unscaled portion of the CCSD correlation energy from opposite-spin or singlet doubles correlations."
+}
+
+cc_properties["ccsd_singles_energy"] = {
+    "type": "number",
+    "description": "The singles portion of the CCSD correlation energy. Zero except in ROHF."
+}
+
+cc_properties["ccsd_doubles_energy"] = {
+    "type":
+    "number",
+    "description":
+    "The doubles portion of the CCSD correlation energy including same-spin and opposite-spin correlations."
+}
+
+cc_properties['ccsd_total_correlation_energy'] = {
+    "type":
+    "number",
+    "description":
+    "The total CCSD correlation energy."
+}
+
+cc_properties['ccsd_total_energy'] = {
+    "type":
+    "number",
+    "description":
+    "The total CCSD energy (CCSD correlation energy + HF energy)."
+}
```

Comment:
general comment:

seems is good accordance with https://github.com/qcdb/qcdb/blob/master/qcdb/qcvars/psivardefs.py#L93-L101

Should we get equations connected with this doc _now_, as eqs. are so much simpler than words for these expressions?

Reply:
How would you encode it, delimited LaTeX (`$$\hat{H}\psi = E\psi$$`)?

Reply:
No, I've had enough of unenforceable equations. Maybe add accession numbers to expressions like [these](https://github.com/qcdb/qcdb/blob/master/qcdb/qcvars/psivardefs.py#L96-L98) and reference them in the schema descriptions.